### PR TITLE
feat: Add magazine section

### PR DESCRIPTION
**Pull request by ValGeorgiev**
Added magazine section with magazine cover image.

Mobile Portrait:
image

Tablet Portrait:
image

Tablet modal view:
image

### CODE REVIEW

**Comment by tuncaulubilge** [comment]
@ValGeorgiev can you add magazine cover into the section-fragment query, and add a story to section storybook for a magazine section?

Also it'd be nice to use an actual magazine cover image taken from TPA on screenshots and storybooks

File: packages/section/src/magazine-cover.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    const { cover, breakpoint, publicationName } = this.props;
+
+    const ratio = getRatio(cover, breakpoint);
+    const uri = getImageUri(cover, breakpoint);
+    const imageHeight = (height - NavBarHeight) * 0.65;
+    return (
+      <View style={[styles.container]}>
+        <View style={styles.header}>
+          <Text style={styles.title}>
+            {publicationName === "TIMES" ? "T" : "ST"} Magazine
```
Comment (nehasri89) [inline packages/section/src/magazine-cover.js]: As discussed we need to check for this data if it feeds in from tpa

Comment (tuncaulubilge) [inline packages/section/src/magazine-cover.js]: Magazine title is a part of the original images fed by editorial, so we don't need a separate view in there for a title

**Comment by tuncaulubilge** [comment]
We also need to add `StandardSlice` to section-fragment query, but you can do that in a separate PR

**Comment by tuncaulubilge** [comment]
We also still need the cover to be added to the section-fragment query as well. You need to ask for all the crops and use the first one (there'll be only one anyway)
```
fragment magazineSection on MagazineSection {
  name: __typename
  cover {
    crops {
      ratio
      url
    }
  }
```

File: packages/provider-queries/src/section-fragment.js [hunk 2]
```diff
@@ -39,6 +39,24 @@ export default gql`
...
+            }
+            listingAsset {
+              ...listingAsset32
+            }
+          }
+          headline
+          leadAsset {
+            ...leadAsset32
+          }
+          strapline
```
Comment (tuncaulubilge) [inline packages/provider-queries/src/section-fragment.js]: Do we need a strapline?

Comment (ValGeorgiev) [reply packages/provider-queries/src/section-fragment.js]: good catch, removed

File: packages/section/src/magazine-cover.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  render() {
+    const { height } = Dimensions.get("window");
+    const imageHeight = height * 0.65;
+    const { breakpoint } = this.props;
+
+    const styles = styleFactory(breakpoint);
+
+    return (
+      <View style={[styles.container]}>
+        <TouchableOpacity
```
Comment (tuncaulubilge) [inline packages/section/src/magazine-cover.js]: Can you use our Link package for clickables?

Comment (ValGeorgiev) [reply packages/section/src/magazine-cover.js]: sure

**Comment by tuncaulubilge** [comment]
What's going on with the `yarn.lock`?

**Pull request merged**